examples/Example/src/prot_snap/index.tsx

```diff
@@ -0,0 +1,347 @@
+import React, { useState } from 'react';
+import { Alert, StyleSheet, useWindowDimensions } from 'react-native';
+import { GestureMonitor, Gesture } from 'react-native-gesture-handler';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  runOnJS,
+} from 'react-native-reanimated';
+import { useAnimatedGesture } from '../useAnimatedGesture';
+
+const filters = ['red', 'green', 'blue', 'yellow', 'orange', 'cyan'];
+const MAX_VIDEO_DURATION = 60000;
+
+export default function Home() {
+  const filter = useSharedValue(0);
+  const filterOffset = useSharedValue(0);
+  const zoom = useSharedValue(1);
+  const [scale, setScale] = useState(1);
+  const [selectedFilter, setSelectedFilter] = useState(0);
+  const [recording, setRecording] = useState(false);
+  const [remainingTime, setRemainingTime] = useState(MAX_VIDEO_DURATION);
+  const [recordingInterval, setRecordingInterval] = useState(-1);
+
+  const filtersPanGesture = Gesture.pan()
+    .setOnUpdate((e) => {
+      'worklet';
+      filter.value =
+        filter.value + (filterOffset.value - e.translationX) * 0.01;
+      filterOffset.value = e.translationX;
+
+      runOnJS(updateSelectedFilter)();
+    })
+    .setOnEnd(() => {
+      'worklet';
+      filterOffset.value = 0;
+      runOnJS(stopFilterScroll)();
+    });
+
+  const buttonTapGesture = Gesture.tap()
+    .setMaxDistance(3)
```

Comment:
Does tap need `maxDistance` to be set?

Reply:
No, it does not. I removed it in baadf962c40f3a3b7abd33dd4fb952961234a539 to avoid confusion